draft-ietf-scitt-architecture.md

```diff
@@ -614,103 +624,103 @@ The retrieved Receipt may be embedded in the corresponding COSE_Sign1 document i
+If a Transparency Service is honest, then a Transparent Statement including a correct Receipt ensures that the Transparent Statement passed its Registration Policy and was recorded appropriately.
 
-Conversely, a corrupt TS may
-1. refuse or delay the registration of Claims;
-2. register Claims that do not pass its Registration Policy (e.g. Claims with Issuer identities and signatures that do not verify.)
-3. issue verifiable Receipts for Claims that do not match its Registry;
-4. refuse access to its Registry (e.g. to Auditors, possibly after storage loss)
+Conversely, a corrupt Transparency Service may
+1. refuse or delay the registration of Signed Statements,
+2. register Signed Statements that do not pass its Registration Policy (e.g., Signed Statement with Issuer identities and signatures that do not verify),
+3. issue verifiable Receipts for Signed Statements that do not match its Registry, or
+4. refuse access to its Registry (e.g., to Auditors, possibly after storage loss).
 
-An Auditor granted (partial) access to the Registry and to a collection of disputed Receipts will be able to replay it, detect any invalid Registration (2) or incorrect  receipt in this collection (3), and blame the TS for them. This ensures any Verifier that trust at least one such Auditor that (2,3) will be blamed to the TS.
+An Auditor granted (partial) access to a Registry and to a collection of disputed Receipts will be able to replay it, detect any invalid Registration (2) or incorrect Receipt in this collection (3), and blame the Transparency Service for them. This ensures any Verifier that trusts at least one such Auditor that (2,3) will be blamed to the Transparency Service.
 
 Due to the operational challenge of maintaining a globally consistent append-only Registry,
-some TS may provide limited support for historical queries on the Claims they have registered,
+some Transparency Services may provide limited support for historical queries on the Transparent Statements they have registered,
 and accept the risk of being blamed for inconsistent Registration or Issuer equivocation.
 
 Verifier and Auditors may also witness (1,4) but may not be able to collect verifiable evidence for it.
 
-**Availability of Transparent Signed Claims**
+#### Availability of Transparent Signed Statement
 
 Networking and Storage are trusted only for availability.
 
-Auditing may involve access to data beyond what is persisted in the TS log. For example, the registered TS may include only the hash of a detailed SBOM, which may limit the scope of auditing.
+Auditing may involve access to data beyond what is persisted in the Transparency Services. For example, the registered Transparency Service may include only the hash of a detailed SBOM, which may limit the scope of auditing.
```

Comment:
Auditing isn't normally regarded as a mechanism for establishing or ensuring availability. Should this be in a different section?